swarm/api/http/server.go

```diff
@@ -189,7 +189,9 @@ func (s *Server) HandleBzzGet(w http.ResponseWriter, r *http.Request) {
 		if err != nil {
 			RespondError(w, r, fmt.Sprintf("Had an error building the tarball: %v", err), http.StatusInternalServerError)
```

Comment:
please add an empty return line after this line, then the block which you added would not be necessary.
in fact, there should be a return statement there (it slipped below my radar in the `http` package refactoring), the code below it is correct and should stay as is.

Reply:
oops, sorry, I missed your reply. Glad it's already fixed.